build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -x
+
+# Install Prerequisites
+
+# 0. Setup Python
+
+wget https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/python-3.5.tar.bz2
+tar xjf python-3.5.tar.bz2 --directory /
+source ~/virtualenv/python3.5/bin/activate
```

Comment:
Assumes presence of a virtual environment with python 3.5. Could you clarify how the fresh download will be used in this command. Maybe worth while to also check if python 3.5 is already available.